<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>Haffman 压缩/解压 - Qt 版</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">

    <item>
     <layout class="QHBoxLayout" name="inputLayout">
      <item>
       <widget class="QLabel" name="inputLabel">
        <property name="text">
         <string>输入文件路径（待压缩文本 / 待解压 zip）：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputFileLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="browseInputFileButton">
        <property name="text">
         <string>浏览...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>

    <item>
     <layout class="QHBoxLayout" name="outputLayout">
      <item>
       <widget class="QLabel" name="outputLabel">
        <property name="text">
         <string>输出目录（可手动输入，不存在会自动创建）：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outputDirLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="browseOutputDirButton">
        <property name="text">
         <string>浏览...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>

    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="compressButton">
        <property name="text">
         <string>压缩文本文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="decompressButton">
        <property name="text">
         <string>解压 zip 文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>

    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>

   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
